benchmarks/gbench/mhp/CMakeLists.txt

```diff
@@ -18,10 +18,14 @@ add_executable(
   stencil_1d.cpp
   stencil_2d.cpp
   chunk.cpp
-  mdspan.cpp
+  # mdspan.cpp
   mpi.cpp)
 # cmake-format: on
 
+if(NOT ENABLE_SYCL)
```

Comment:
add jira issue about this ("workstream") and write a comment here why `mdspan.cpp` is just in non-sycl config + write jira id in this comment